XMonad/Layout/Magnifier.hs

```diff
@@ -39,6 +42,8 @@ module XMonad.Layout.Magnifier
 
       -- * Messages and Types
       MagnifyMsg (..),
+      MagnifyThis(..),
+      Start(..),
```

Comment:
I'm a bit concerned about this: `Start`, `On`, `Off` don't seem like good/safe names to export. Also, I'm not sure if renaming this for easier understanding of the "constructor" function is worth having a name that's confusing in the actual implementation, where it signifies the current state, not initial state.

I'd almost think that consistently naming the record fields, just using Bool for this one, and exporting the fields may be better long term. We may add a default instance and not export the masterWins field, but note that both Tall and ResizableTall make it possible to set the initial number of masters, so we actually do want to export this one too!

(And then it becomes questionable whether the `magnify` function is worth it.)

Reply:
Good point, thanks.  `On` and `Off` are probably a little bit too generic to export; using bool and then translating to the internal names sounds good to me (I do think it's nice that we have proper names for this internally and are not just using bool).

I'm kind of ambivalent about just exporting all of `Magnify`.  Best case we can deprecate almost all functions and tell people to use the data constructor directly instead (like X.L.Spacing essentially did), worst case there is a pressing need to change the internals for some new feature or fix and we can't do it because it'll break people's configurations.

Reply:
I don't have any strong preference about (not) exporting either, so… can't help you there, or at least not immediately. :-)